setup.cfg

```diff
@@ -89,6 +89,7 @@ docs =
     scipy>=1.1
     matplotlib>=3.1,!=3.4.0
     sphinx-changelog>=1.1.0
+    sphinx-reredirects
```

Comment:
I guess this package is from https://gitlab.com/documatt/sphinx-reredirects ? Are we worried about single-point-of-failure here, in that the author will abandon this package and then we end up with a bunch of broken links?

I think it is okay to introduce a new dependency here instead of a new PR. But I want to make sure we are not using someone's random pet project that will get abandoned.

Reply:
Perhaps a less controversial approach to let this PR get merged sooner than later is, as you said, introduce this dependency in a follow-up PR. In the meantime, instead of deleting the ref to `docs/development/workflow/virtualenv_detail.rst`, move that ref somewhere in the new page, or keep the old page with the ref, but scrub out the content and point to the new page without this reredirect thingy.

Reply:
I think it's pretty low-risk.  The developer seems open to accepting patches, and the extension itself is about 100 lines so if it falls into disrepair, could easily be merged into sphinx-astropy or the like.

Otherwise I don't think this is worth expending any more effort on.

Reply:
Let's see what others say.